datadog_checks_base/datadog_checks/base/utils/common.py

```diff
@@ -1,6 +1,8 @@
 # (C) Datadog, Inc. 2018
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
+from __future__ import division
```

Comment:
just out of curiosity, how is division different in different python versions?

Reply:
```
ofek@ozone ~\Desktop $ python -c "print(3 / 2)"
1.5
ofek@ozone ~\Desktop $ python2.7 -c "print(3 / 2)"
1
```

Reply:
To floor the result do `//`